config/settings/base.py

```diff
@@ -113,7 +113,7 @@
 
 LANGUAGE_CODE = 'en-us'
 
-TIME_ZONE = 'UTC'
+TIME_ZONE = 'America/Phoenix'
```

Comment:
This will fix the "server" timezone, which for all intents and purposes is just applicable to us. The user never sees the order datetime, and we don't care what their local time is, just when it was purchased relative to us.